### PR TITLE
PIM-422 refactor into small methods

### PR DESCRIPTION
**Pull request by Trimikraj**

### CODE REVIEW

**Comment by Expressionless** [comment]
LGTM

**Review by Expressionless — APPROVED** [review]
LGTM

**Review by varunshastry — APPROVED** [review]

File: src/main/java/life/genny/messages/managers/QSendGridMessageManager.java [hunk 1]
```diff
@@ -130,6 +132,11 @@ public void sendMessage(BaseEntityUtils beUtils, BaseEntity templateBe, Map<Stri
 			mail.setTemplateId(templateId);
 			mail.setFrom(from);
 
+			System.out.println("Mail: " + mail);
```
Comment (varunshastry) [inline src/main/java/life/genny/messages/managers/QSendGridMessageManager.java]: s.o.p to be replaced with logger.

**Review by varunshastry — CHANGES_REQUESTED** [review]

File: src/main/java/life/genny/messages/managers/QSendGridMessageManager.java (hunk 1 above)
Comment (Trimikraj) [reply src/main/java/life/genny/messages/managers/QSendGridMessageManager.java]: done

**Review by varunshastry — APPROVED** [review]

**Pull request merged**